tests/ci/integration/run_sslproxy_integration.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  - SRC_ROOT(aws-lc)
+#    - SCRATCH_FOLDER
+#      - sslproxy
+#      - AWS_LC_BUILD_FOLDER
+#      - AWS_LC_INSTALL_FOLDER
+#      - SSLPROXY_BUILD_FOLDER
+#      - libevent
+#      - LIBEVENT_INSTALL_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER="${SRC_ROOT}/SSLPROXY_BUILD_ROOT"
+SSLPROXY_SRC_FOLDER="${SCRATCH_FOLDER}/sslproxy"
+SSLPROXY_BUILD_FOLDER="${SCRATCH_FOLDER}/sslproxy-aws-lc"
+SSLPROXY_PATCH_FOLDER="${SRC_ROOT}/tests/ci/integration/sslproxy_patch"
+AWS_LC_BUILD_FOLDER="${SCRATCH_FOLDER}/aws-lc-build"
+AWS_LC_INSTALL_FOLDER="${SCRATCH_FOLDER}/aws-lc-install"
+LIBEVENT_SRC_FOLDER="${SCRATCH_FOLDER}/libevent"
+LIBEVENT_INSTALL_FOLDER="${SCRATCH_FOLDER}/libevent-install"
+
+mkdir -p ${SCRATCH_FOLDER}
+rm -rf ${SCRATCH_FOLDER}/*
+cd ${SCRATCH_FOLDER}
+
+function libevent_install() {
```

Comment:
I never knew libevent dependened on / supported OpenSSL specific features.

Reply:
Yeah me neither. Apparently it's configurable, and they support LibreSSL as well.